src/base/backend_intf.ml

```diff
@@ -1,51 +1,11 @@
-module type Cvar_intf = sig
-  type field
-
-  type t [@@deriving sexp]
-
-  val length : t -> int
```

Comment:
Nit: this is removed later. Agreed it says it is for debugging purposes, but I would like trying to stay atomic in the commits, and it would have been nice to have a separate commit that says it removes the length value. I know, picky :smile: